#VRML_SIM R2021b utf8
WorldInfo {
  coordinateSystem "NUE"
}
Viewpoint {
  orientation -0.45936854311705727 -0.8799837185356792 -0.12086850998763338 0.6278346240012853
  position -1.2057486128914356 1.147788966465248 2.1452652614668537
}
TexturedBackground {
}
TexturedBackgroundLight {
}
RectangleArena {
  translation 0 -0.01 0
}
Robot {
  rotation 0 1 0 3.14
  children [
    DEF Body Transform {
      translation 0 0.0415 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0 0.666667 1
          }
          geometry Cylinder {
            height 0.08
            radius 0.045
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor 0.045 0.025 0
      }
      device [
        DEF motor1 RotationalMotor {
          name "motor1"
        }
        PositionSensor {
          name "ps1"
        }
      ]
      endPoint Solid {
        translation 0.045 0.025 0
        rotation 0 0 1 1.57
        children [
          DEF Wheel Shape {
            appearance PBRAppearance {
              baseColor 0.988235 0.286275 0.164706
              roughness 1
              metalness 0
            }
            geometry Cylinder {
              height 0.01
              radius 0.025
            }
          }
        ]
        boundingObject USE Wheel
        physics DEF Wheel_physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor -0.045 0.025 0
      }
      device [
        DEF motor2 RotationalMotor {
          name "motor2"
        }
        PositionSensor {
          name "ps2"
        }
      ]
      endPoint Solid {
        translation -0.045 0.025 0
        rotation 0 0 1 1.57
        children [
          USE Wheel
        ]
        name "solid(1)"
        boundingObject USE Wheel
        physics USE Wheel_physics
      }
    }
    Solid {
      translation 0.015 0.07 0.045
      children [
        DEF eye Shape {
          appearance PBRAppearance {
            baseColor 1 0.333333 0
            roughness 1
            metalness 0
          }
          geometry Box {
            size 0.005 0.005 0.005
          }
        }
      ]
      name "solid(2)"
    }
  ]
  boundingObject USE Body
  physics Physics {
  }
  controller "drive_my_bot"
}
